remove highly from highly recommend (#3437)
camunda/camunda-docs: docs/components/connectors/out-of-the-box-connectors/amazon-eventbridge.md

Before using the **Amazon EventBridge Connector**, ensure you have the necessary permissions in your AWS account to send events to EventBridge. You will need an access key and secret key of a user with the appropriate permissions. Refer to the [AWS documentation](https://docs.aws.amazon.com/eventbridge/latest/userguide/auth-and-access-control-eventbridge.html) for more information.

:::note

> We do not recommend exposing your AWS IAM credentials as plain text. Instead, use Camunda secrets. Refer to our documentation on [managing secrets](/components/console/manage-clusters/manage-secrets.md) to learn more.

Reviewer: I could see this rewritten as the following:

```suggestion
Use Camunda secrets to avoid exposing your AWS IAM credentials as plain text. Refer to our documentation on [managing secrets](/components/console/manage-clusters/manage-secrets.md) to learn more.
```

Or even remove the second line and link to the page via the first sentence.
